app/models/concerns/liked.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Liked
+  def liked_by(user)
+    likes.find_or_create_by(user: user)
+  end
+
+  def unliked_by(user)
+    like = likes.find_by(user: user)
+    like.destroy if like.present?
```

Comment:
__[reek]__
* [FeatureEnvy] refers to 'like' more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)

<!-- Comment by [SideCI](https://sideci.com) -->